data_structures/linked_lists/circular_linked_list.rb

```diff
@@ -0,0 +1,105 @@
+# Define a node for the list
+class Node
+  attr_accessor :value, :next
+
+  def initialize(value)
+    @value = value
+    @next = nil
+  end
+end
+
+# Class for circular linked list (last node points to the head node)
+class CircularList
+  attr_accessor :head
+
+  def initialize
+    @head = nil
+  end
+
+  def insert_end(value)
```

Comment:
(just to be sure) Is this method intentionally named `insert_end` rather than `insert_tail`?

I'm not familiar with terms about linked lists but [circular_linked_list.py of TheAlgorithms/Python](https://github.com/TheAlgorithms/Python/blob/f66325a01bd790107c535f922aa0d9cfd75935e3/data_structures/linked_list/circular_linked_list.py) uses the names `insert_tail` and `delete_tail`.

Reply:
We can do `insert_tail` to keep it consistent with Python. No objections on my end.

Reply:
Thanks for response!

And sorry, I didn't write well. What I was concerned was that [`CircularList#insert_end`](https://github.com/TheAlgorithms/Ruby/blob/c99c8ffb04e1939e9ee30fd82993aaff74263f9a/data_structures/linked_lists/circular_linked_list.rb#L19) and [`CircularList#delete_tail`](https://github.com/TheAlgorithms/Ruby/blob/c99c8ffb04e1939e9ee30fd82993aaff74263f9a/data_structures/linked_lists/circular_linked_list.rb#L74) look inconsistent, not about consistency with Python.

Anyway it is resolved at 906ccac28d03b358e8fa86e6ab3db7894967ea59, so I'll mark this one as resolved. But feel free to replace `tail` with `end` if you like it :).

Reply:
I'm comfortable with both terms as long as all the linked lists algorithms in the Ruby repo use the same interface methods.